Build/Build.cs

```diff
@@ -208,10 +230,33 @@ class Build : NukeBuild
                 .AddFileFilters("-*.g.cs")
                 .SetAssemblyFilters("+FluentAssertions"));
 
-            string link = TestResultsDirectory / "reports" / "index.html";
+            string link = CoverageResultDirectory / "index.html";
             Information($"Code coverage report: \x1b]8;;file://{link.Replace('\\', '/')}\x1b\\{link}\x1b]8;;\x1b\\");
         });
 
+    Target Coveralls => _ => _
+        .OnlyWhenDynamic(() => !string.IsNullOrWhiteSpace(CoverallsToken) && (RunAllTargets || HasSourceChanges))
+        .DependsOn(CodeCoverage)
+        .Consumes(CodeCoverage)
+        .Executes(() =>
+        {
+            CoverallsNet(s => s
+                .SetDryRun(IsLocalBuild)
+                .SetProcessArgumentConfigurator(x => x
+                    .Add("--lcov"))
+                .SetInput(CoverageResultDirectory / "lcov.info")
+                .SetRepoToken(CoverallsToken)
+                .When(IsPullRequest,
+                    s => s.SetPullRequest(GitHubActions?.PullRequestNumber)
+                )
+                .EnableUserRelativePaths()
```

Comment:
```suggestion
                .EnableUserRelativePaths()
                .SetJobId(GitHubActions?.RunId)
```

Reply:
This should do the trick

Reply:
Hmm.. `.SetJobId` accepts only an `int?` but currently the github run ids are beyond `int.MaxValue`..